mne/event.py

```diff
@@ -187,8 +188,10 @@ def _read_events_fif(fid, tree):
     event_list = event_list.reshape(len(event_list) // 3, 3)
     return event_list, mappings
 
-
-def read_events(filename, include=None, exclude=None, mask=0):
+@deprecated("The default setting for the argument 'mask_type' will change "
+            "from None to 'and' in v0.14.")
```

Comment:
got it. but now since the default is None, this will raise, and any test counting the number of warnings that arise (what's the utility of the count if it's a warning?) will break